__tests__/app.test.js

```diff
@@ -210,4 +211,56 @@ describe("PATCH /api/articles/:article_id", () => {
             expect(body.msg).toBe("article not found!")
         })
     })
+})
+describe("/api/articles(queries)", () => {
+    test("GET 200 - filters the articles by specified topic", () => {
+        return request(app).get("/api/articles?topic=mitch").expect(200).then(({body}) => {
+            expect(body.articles).toHaveLength(11)
+            body.articles.forEach((article) => {
+                expect(article.topic).toBe("mitch")
+            })
+        })
+    })
+    test("GET 200 - given no topic query returns all the articles", () => {
```

Comment:
You still need a test for a valid topic that has no associated articles - i.e. valid topic, status 200, returns an empty array.